### ISSUE TITLE
Not available on google play store anymore?

### ISSUE THREAD

**Issue by randrcomputers** [thecowan/wallpanel-android#324]
Is this app still available via play store? or can we only side load it now? cannot find in app store to install on a new table.

**Comment by RogerMCT** [comment]
I'm wondering the same thing.  I can't find it on Amazon or Google stores.

**Comment by randrcomputers** [comment]
yea very strange. I hope this app is still supported

**Comment by lesykm** [comment]
cc @thanksmister as some who has more info (was maintainer in the past).

**Comment by thanksmister** [comment]
The reason the application is no longer available is because there is a new maintainer.  The new maintainer needs to publish the app again to the stores with his own keys.  I have transferred the Google store page to @thecowan, the new maintainer.  Amazon requires a new account for developers to publish an app.

**Comment by javawizard** [comment]
Makes sense. For those of us looking to install WallPanel in the mean time, what's the best way to do it?

**Comment by randrcomputers** [comment]
@thecowan if you need help publishing i have a dev account and could upload it for you.

**Comment by chris-burrow-apps** [comment]
Hi. When will this be published to the Google Play Store? Been a little while since last update

**Comment by AzureAzim** [comment]
I am also no longer seeing this in the 'Managed Google Play Store'
I use this to push wall panel to kiosk devices managed by an MDM, but its now no longer pushing due to being missing from the managed store

**Comment by AzureAzim** [comment]
> 

@randrcomputers  is it possible to rebuild this from source and repost it under your dev account? This is blocking me from automating my deployment across multiple devices.

**Comment by randrcomputers** [comment]
Yes I could easily publish it but not mine to publish. Let’s hope current dev finds time to do it. You can easily just install .apk manually until then.
